modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -2321,6 +2321,14 @@ public boolean equals(Object obj) {
         return false;
     }
```

Comment:
in the case of FXMLoader, hashCode() also follows the logic of equals().
However, we could mix something specific to FXMLoader to the hash, to avoid collision with origin URL.

Reply:
I think this is fine as you now have it. If anyone feels it is worth it, we could file a follow-on issue to look into whether `equals` should be changed, but I don't know whether it is worth it.